fontbe/src/post.rs

```diff
@@ -21,7 +22,9 @@ impl Work<Context, Error> for PostWork {
         // TODO optionally drop glyph names with format 3.0.
         // TODO a more serious post
         let static_metadata = context.ir.get_final_static_metadata();
-        let post = Post::new_v2(static_metadata.glyph_order.iter().map(|g| g.as_str()));
+        let mut post = Post::new_v2(static_metadata.glyph_order.iter().map(|g| g.as_str()));
+        post.underline_position = FWord::new(static_metadata.misc.underline_position.0 as i16);
+        post.underline_thickness = FWord::new(static_metadata.misc.underline_thickness.0 as i16);
```

Comment:
ufo2ft does otRound, does not truncate. In general we use that when converting stuff that in ufo is allowed to be a plist "real" to an integer.